docs/config-local-settings.rst

```diff
@@ -363,6 +363,12 @@ CARBONLINK_TIMEOUT
 
   Timeout for carbon-cache cache queries in seconds.
 
+CARBONLINK_HASHING_TYPE
+  `Default: carbon_ch`
+
+  `Possible values: carbon_ch, fnv1a_ch`
+
+  The default `carbon_ch` is Graphite's traditional consistent-hashing implementation. Alternatively, you can use the `fnv1a_ch`, which supports the Fowler–Noll–Vo hash function (FNV-1a) hash implementation offered by the `carbon-c-relay relay <https://github.com/grobian/carbon-c-relay>`_ project.
```

Comment:
Sorry, one more fix (my fault)... s/you can use the/you can use/. Also in the settings example below. Thanks!